Formula/kube-score.rb

```diff
@@ -0,0 +1,35 @@
+class KubeScore < Formula
+  desc "Kubernetes object analysis recommendations for improved reliability and security"
+  homepage "https://kube-score.com"
+  url "https://github.com/zegl/kube-score/archive/refs/tags/v1.13.0.tar.gz"
+  sha256 "a09198f9a49c0008f9a34b418d85d3eb1299d0f04077e423eff30f1cad8abd43"
+  license "MIT"
+  head "https://github.com/zegl/kube-score.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "./cmd/kube-score"
+  end
+
+  test do
+    (testpath/"test.yaml").write <<~EOS
```

Comment:
since we're setting `ldflags` above for version etc, let's add a test to catch if/when those flags change
```suggestion
    assert_match version.to_s, shell_output("#{bin}/kube-score version")
    
    (testpath/"test.yaml").write <<~EOS
```